apps/api/src/server/repositories/representation.repository.js

```diff
@@ -665,29 +665,49 @@ export const getApplicationRepresentationForDownload = async (caseId, skip, batc
  * @param {number} caseId
  * @returns {PrismaPromise<GetFindResult<Prisma.RepresentationSelect>[]>}
  */
-export const getPublishableRepresentations = async (caseId) =>
-	databaseConnector.representation.findMany({
-		select: {
-			id: true,
-			reference: true,
-			status: true,
-			redacted: true,
-			received: true,
-			represented: {
-				select: {
-					firstName: true,
-					lastName: true,
-					organisationName: true
-				}
-			}
-		},
+export const getPublishableRepresentations = async (caseId) => {
+	const totalPublishableRepsCount = await databaseConnector.representation.count({
 		where: {
 			caseId,
 			OR: [{ status: 'PUBLISHED', unpublishedUpdates: true }, { status: 'VALID' }]
-		},
-		orderBy: [{ status: 'desc' }, { reference: 'asc' }]
+		}
 	});
 
+	let batchSize = 2000;
+	let publishableRepsPromises = [];
+
+	for (let i = 0; i < totalPublishableRepsCount; i += batchSize) {
+		const promise = databaseConnector.representation.findMany({
```

Comment:
Nitpicky: Can probably push the promise directly into the array or call the promise a more descriptive name